common/mempool.go

```diff
@@ -105,35 +104,27 @@ func refreshMempoolTxns() error {
 			// We've already fetched this transaction
 			continue
 		}
-		g_txidSeen[txid(txidstr)] = struct{}{}
+
 		// We haven't fetched this transaction already.
+		g_txidSeen[txid(txidstr)] = struct{}{}
 		txidJSON, err := json.Marshal(txidstr)
 		if err != nil {
 			return err
 		}
-		// The "0" is because we only need the raw hex, which is returned as
-		// just a hex string, and not even a json string (with quotes).
-		params := []json.RawMessage{txidJSON, json.RawMessage("0")}
+
+		params := []json.RawMessage{txidJSON, json.RawMessage("1")}
```

Comment:
An interesting side effect of this (using verbose `getrawtransaction`) is that if we get the list of mempool transactions, and then a new block arrives before we finish sending them, any transactions that got mined will have height set to their mined height instead of 0 (and then once all the new "mempool" txs are sent, the stream will end as the server detects the new block).

Reply:
Good point, @str4d. Also, if the returned transaction has a nonzero height, it would be best to filter it out since we know it's no longer in the mempool (we must have hit this timing window).